forge-gui/res/cardsfolder/upcoming/synthesis_pod.txt

```diff
@@ -0,0 +1,9 @@
+Name:Synthesis Pod
+ManaCost:3 PU
+Types:Artifact
+A:AB$ DigUntil | Cost$ 1 UP T ExileFromStack<1/Spell.YouCtrl/spell you control> | ValidTgts$ Opponent | Valid$ Card.cmcEQX | FoundDestination$ Exile | RevealedDestination$ Library | ImprintFound$ True | Shuffle$ True | SubAbility$ DBPlay | SpellDescription$ Target opponent reveals cards from the top of their library until they reveal a card with mana value equal to 1 plus the exiled spell's mana value. Exile that card, then that player shuffles. You may cast that exiled card without paying its mana cost. 
```

Comment:
You can probably use `Card.YouCtrl` for now

Reply:
Yes that made it work, but after further testing it seems that SVar:X:Exiled$CardManaCost/Plus.1 isn't working at all. I guess CostExileFromStack.java needs some additions ?

Reply:
You can try this in payAsDecided:
`ability.addCostToHashList(CardUtil.getLKICopy(sa.getHostCard()), "Exiled");`

Reply:
That made it work ! Thanks for the help as usual 
![image](https://user-images.githubusercontent.com/67333662/214646843-993be4da-b84a-440e-8226-ca15b087daca.png)

Reply:
You should create the LKI as the first thing, not after exiling